forgerock-auth/src/main/java/org/forgerock/android/auth/selfservice/Device.kt

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright (c) 2024 ForgeRock. All rights reserved.
```

Comment:
Do we still use ForgeRock in the copyright header? (I'm not sure...)

Reply:
I believe we should update to `Ping Identity` for any updated file